### PR TITLE
[Do not merge!] opinionated draft suggestion - for reference only

### PR DESCRIPTION
**Pull request by stp-ip**
One proposed draft for the new spec as an overview.
I tried to integrate all discussions and suggestions from PRs, issues and the old spec.
Additionally I tried to abstract away a few specifics to make merging different sections/ideas easier.
I find that it's easier to discuss this as one document at least as an overview for now.
Please provide some feedback and if need be, we could split it out to various PRs afterwards. Especially as this PR is only changing the schema not yet the rest of the code.

Tasks where help is needed:
- [ ] enum for publications.identifiers
- [ ] enum for skills.level
- [ ] enum for languages.level
- [ ] mark required data and optional ones
- [ ] verify the correct usage of additionalProperties on each section
- [ ] should education.courses and education.exams be merged?
- [ ] use bibJSON for publications

### CODE REVIEW

**Comment by stp-ip** [comment]
One proposed draft for the new spec as an overview.
I tried to integrate all discussions and suggestions from PRs, issues and the old spec.
Additionally I tried to abstract away a few specifics to make merging different sections/ideas easier.
I find that it's easier to discuss this as one document at least as an overview for now.
Please provide some feedback and if need be, we could split it out to various PRs afterwards. Especially as this PR is only changing the schema not yet the rest of the code.

Tasks where help is needed:
* [ ] enom for publications.identifiers
* [ ] enom for skills.level
* [ ] enom for languages.level
* [ ] mark required data and optional ones
* [ ] verify the correct usage of additionalProperties on each section

**Comment by stp-ip** [comment]
@jsonresume @aloisdg @francescoes @JMPerez

File: schema.json [hunk 1]
```diff
@@ -90,141 +135,424 @@
...
+						}
+					},
+					"labels": {
+						"type": "object",
+						"$ref": "#/definitions/labels"
+					}
+				}
 			}
 		},
 		"awards": {
```
Comment (stp-ip) [inline schema.json]: Global achievements instead of awards?

Comment (chrisdotcode) [inline schema.json]: :+1: `achievements` is more genericl

File: schema.json [hunk 2]
```diff
@@ -337,43 +721,214 @@
...
+					},
+					"labels": {
+						"type": "object",
+						"$ref": "#/definitions/labels"
+					}
+				}
+			}
+		},
 
+		"certification": {
```
Comment (stp-ip) [inline schema.json]: Certifications needed? Or should that be done under education or achivements?

**Comment by stp-ip** [comment]
Yeah I am just using this PR to have a better overview and mark discussion points to actual schema changes to have a reference point and to not forget things we discussed.
I agree that smaller PRs are the way to go, but a huge PR can give us the ability to look for how things could be merged or separated more easily as we have the whole picture.

**Comment by aloisdg** [comment]
"enom" should be "enum" in your original post

**Comment by stp-ip** [comment]
Thanks. Fixed.

File: schema.json [hunk 3]
```diff
@@ -90,141 +135,424 @@
...
-			  }
-
+				"type": "object",
+				"additionalProperties": true,
+				"properties": {
+					"company": {
+						"type": "string",
+						"description": "e.g. Facebook"
+					},
+					"position": {
```
Comment (TheKevJames) [inline schema.json]: This could also be an array of positions held

Comment (stp-ip) [inline schema.json]: It seems we are going the one role per project route.

**Comment by stp-ip** [comment]
Will close this opinionated draft, as some of the ideas are already deprecated, merged or still being discussed.

**Pull request closed without merging**